src/ts/ConvivaAnalytics.ts

```diff
@@ -494,7 +495,8 @@ export class ConvivaAnalytics {
     this.client.releasePlayerStateManager(this.playerStateManager);
 
     this.sessionKey = Conviva.Client.NO_SESSION_KEY;
-    this.lastSeenBitrate = null;
+    // this.lastSeenBitrate = null;
```

Comment:
Why did you remove the reset of the  `lastSeenBitrate`?

Reply:
After the video is finished you can click on "replay".  If you set lastSeenBitrate to NULL then the bitrate is empty.

Reply:
Hm. What happens in case of a different source 🤔 Is it ensured that when a new source is loaded the `onVideoQualityChanged` is called?